src/pages/benefits/benefitFormUI/BenefitFormUI.tsx

```
@@ -58,74 +73,109 @@
+
+        const docSchemaData = convertDocumentFields(docSchemaArr, userDocs);
+        setDocSchema(docSchemaData);
+        const properties = {
+          ...(applicationSchema.properties || {}),
+          ...(docSchemaData?.properties || {}),
+        };
+        // const properties = applicationFormSchema.properties;
+        const required = Object.keys(properties).filter((key) => {
+          const isRequired = properties[key].required;
+          if (isRequired !== undefined) {
+            delete properties[key].required;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)